src/hotspot/share/prims/nativeLookup.cpp

```diff
@@ -351,16 +344,13 @@ address NativeLookup::lookup_entry(const methodHandle& method, TRAPS) {
   }
 
   // 2) Try JNI long style
-  entry = lookup_style(method, pure_name, long_name, args_size, true,  CHECK_NULL);
+  entry = lookup_style(method, pure_name, long_name, args_size, CHECK_NULL);
   if (entry != nullptr) return entry;
 
   // 3) Try JNI short style without os prefix/suffix
```

Comment:
Please update comment as there is no os prefix/suffix now

Reply:
Actually, it was not just the comment that were wrong, it was the actual code ("if the comment and code don't agree, then most likely both are wrong").

The steps 3 and 4 were just 1 and 2 without the os prefix/suffix, which we do not need anymore. I removed 4 but for some reason I did not realize that I should remove 3 as well. And I kept an unnecessary `if (entry != nullptr) return entry;`...